agent_deploy/kubernetes/sysdig-agent-ebpf-daemonset-v1.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: sysdig-agent
+  labels:
+    app: sysdig-agent
+spec:
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: sysdig-agent
+    spec:
+      volumes:
+      - name: osrel
+        hostPath:
+          path: /etc/os-release
+          type: FileOrCreate
+      - name: dshm
+        emptyDir:
+          medium: Memory
+      - name: docker-sock
+        hostPath:
+          path: /var/run/docker.sock
```

Comment:
Does this mean this is supported only for docker runtimes ?  
CRI/Containerd ?

Reply:
To the best of my knowledge, non-docker runtimes are in beta at the moment for GKE. Am I wrong?

Reply:
@nathan-b , lets include the daemonset mount points with those in the sysdig-agent-daemonset-v2.yaml. docker.sock is no longer a mount point, we instead mount /run and /var/run.

Reply:
@krishnan-ramkumar  nathan is correct containerd is currently beta for GKE cluster deployment ![image](https://user-images.githubusercontent.com/22457772/53375857-6ce70580-392a-11e9-9c96-d5fa26396892.png)